cms/templates/cms/program_page.html

```diff
@@ -5,8 +5,248 @@
 {% block body_class %}template-programpage{% endblock %}
 
 {% block content %}
-<h1>{{ page.title }}</h1>
+    <div class="app-media layout-boxed">
+        <div class="main-content-wrapper">
+            <section class="main-content">
+                <div class="banner-wrapper banner-sub banner-program-1">
+                    <nav class="navbar navbar-default" role="navigation">
+                        <div class="container-fluid">
+                            <div class="navbar-header">
+                                <a class="navbar-brand" href="javascript:void(0)"><img
+                                        src="/static/images/logo-mit-micromasters-whitered@2x.png" width="215"
```

Comment:
Actually we should copy `logo-mit-micromasters-whitered.png` and use `width="215"` and `height="40"` instead